### ISSUE TITLE
Syntax highlighter colours bad on light theme

### ISSUE THREAD

**Issue by sam-keen** [vegaprotocol/frontend-monorepo#678]
The grey is too light, fails a11y check

![Screenshot 2022-06-29 at 16 27 32](https://user-images.githubusercontent.com/2410498/176476116-3d745d06-5726-4492-87cc-0d8f7f88c14f.png)
